CHANGELOG.md

```diff
@@ -1,5 +1,21 @@
 # Changelog
 
+# 0.7.3 (2019-03-22)
+
+## New Features
+
+- **M3DB**: Add an AllQuery index query type (#1478)
+
+## Bug Fixes
+
+- **M3DB**: Fix to annotation pooling (#1476)
+- **M3Coordinator**: Only panics log a stacktrace rather than expected errors (#1480)
+
+## Performance
+
+- **M3DB**: Use a single results object for merging postings lists across blocks, rather than creating a result per block and discarding them immediately thereafter (#1474)
```

Comment:
Can you say "This should reduce memory utilization significantly for queries that look back over a large number of blocks"